blind75/two_sum.py

```diff
@@ -59,6 +59,17 @@ def solution3(self, nums: List[int], target: int) -> List[int]:
                 if nums[i] + nums[j] == target:
                     return [i,j]
         return []
+    
+    # Time complexity: O(n2), Space complexity: O(1)
+    def solution3(self, nums: List[int], target: int) -> List[int]:
+        if len(nums) == 0:
```

Comment:
Remove the redundant check for an empty list at the beginning of the method. This check is unnecessary because the subsequent for loop effectively handles the case when `nums` is empty by not iterating at all. [important]